projects/media-viewer/src/lib/annotations/annotation-set/annotation/comment/comment.component.ts

```diff
@@ -102,4 +103,9 @@ export class CommentComponent {
     this.updated.emit(this.fullComment);
     this.editable = false;
   }
+
+  private getFirstNonNullParentProperty(el: Node, property: string) {
+    return el.parentNode ? ( el.parentNode[property] ?
```

Comment:
I'm not sure if this is better....

`return !el.parentNode ? null : el.parentNode[property] || this.getFirstNonNullParentProperty(el.parentNode, property);`

Reply:
OK

Reply:
ok